utils/brand.env

```diff
@@ -1,5 +1,5 @@
 export SEARXNG_URL=''
 export SEARXNG_PORT='8888'
 export SEARXNG_BIND_ADDRESS='127.0.0.1'
-export GIT_URL='https://github.com/searxng/searxng'
```

Comment:
ping @return42 @Bnyro

Reply:
Can't we just add that file tp the gitignore? :(

Reply:
This file is needed if you ([like me](https://github.com/searxng/searxng/compare/master...return42:searxng:darmarit.org)) maintain a [brand](https://docs.searxng.org/admin/settings/settings_brand.html) .. its sourced by the [`/utils/searxng.sh`](https://docs.searxng.org/utils/searxng.sh.html).

- [installation](https://docs.searxng.org/admin/installation-scripts.html) and [update](https://docs.searxng.org/admin/update-searxng.html) SearXNG is based on
- [`make buildenv`](https://docs.searxng.org/dev/makefile.html#make-buildenv)
- https://docs.searxng.org/dev/contribution_guide.html#deploy-on-github-io

For those who run a docker instance this file may be annoying in the development phase, for those of us who run a normal installation (without docker) this file is elementary.

Please adjust the git workflow so that only those files are added to the commit where there are changes that belong to the patch / don't just add everything with `git add` without taking a closer look.

[Commits are a firsthand historical record of exactly how and why each line of code came to be. They even come with human-readable messages! As a result, **a repository’s commit history is the best tool a developer can use to explain and understand code** .... Commits are more than just the code they contain. Despite there being no shortage of jokes about them, **commit messages are an extremely valuable – but often overlooked – component of a commit.** Most importantly, they’re an opportunity to speak directly to your audience and explain a change in your own terms.](https://github.blog/2022-06-30-write-better-commits-build-better-projects/)